orchestra/tests/test_dashboard.py

```diff
@@ -470,11 +472,12 @@ def test_assignment_snapshot_upgrades(self):
         good_snapshots = {
             'snapshots': [{
                 'data': {},
-                'datetime': datetime.utcnow().isoformat(),
+                'datetime': (
+                    datetime.utcnow().replace(tzinfo=tzutc()).isoformat()),
                 'type': TaskAssignment.SnapshotType.SUBMIT,
                 'work_time_seconds': 1
             }],
-            '__version': 1}
+            '__version': 2}
```

Comment:
the version helps us do migrations on the data.  see `orchestra/utils/assignment_snapshots.py`